js/modules/k6/http/http_request.go

```diff
@@ -50,6 +51,12 @@ import (
 	null "gopkg.in/guregu/null.v3"
 )
 
+// ErrHTTPForbiddenInInitContext is used when a http requests was made in the init context
+var ErrHTTPForbiddenInInitContext = errors.New("Making http requests in the init context is not supported")
+
+// ErrBatchForbiddenInInitContext is used when batch was made in the init context
+var ErrBatchForbiddenInInitContext = errors.New("Using batch in the init context is not supported")
```

Comment:
Not sure if it's worth it to distinguish between `http.batch()` and `http.request()` errors, reusing the first errors should be fine, after all it will be used for `http.get()`, `http.post()` and all of the rest as well.

Reply:
I was wondering the same thing but decided that batch is different enough, although I am not so sure now